README.md

```diff
@@ -76,6 +76,8 @@ This repository contains algorithms and data structures implemented in C# for ed
 		* [A000027 Natural](./Algorithms/Sequences/NaturalSequence.cs)
 		* [A000040 Primes](./Algorithms/Sequences/PrimesSequence.cs)
 		* [A000045 Fibonacci](./Algorithms/Sequences/FibonacciSequence.cs)
+		* [A007318 Binomial](./Algorithms/Sequences/BinomialSequence.cs)
+		* [A000142 Factorial](./Algorithms/Sequences/FactorialSequence.cs)
```

Comment:
Please sort them by their number

Reply:
@siriak sequences in readme are sorted

Reply:
Sequences are updated, they are infinite for the moment, in tests .Take(n) from LINQ is used